.storybook/themes/commerce.scss

```diff
@@ -803,6 +803,75 @@
 					// for featured enabled, this will apply for the desktop growing to expand full width. On mobile, it doesn't hurt.
 					flex-basis: 100%,
 				),
+				product-gallery-focus-view: (
+					position: fixed,
+					background-color: var(--global-white),
+					height: 100vh,
+					left: 0,
+					overflow: hidden,
+					top: 0,
+					width: 100vw,
+				),
+				product-gallery-focus-view-main-images: (
+					gap: var(--global-spacing-7),
+					height: 100vh,
+					max-width: 100%,
+					overflow-y: auto,
+				),
+				product-gallery-focus-view-main-image: (
+					height: auto,
+					max-width: 100%,
+				),
+				product-gallery-focus-view-toolbar: (
+					background: transparent,
+					bottom: 0,
+					position: fixed,
+					padding-top: var(--global-spacing-5),
+					padding-right: var(--global-spacing-5),
+					padding-bottom: var(--global-spacing-7),
+					right: 0,
+					top: 0,
+				),
+				product-gallery-focus-view-thumbnail-bar: (
+					gap: var(--global-spacing-5),
+					padding-top: var(--global-spacing-7),
+					padding-right: var(--global-spacing-6),
+					padding-bottom: var(--global-spacing-7),
+					padding-left: var(--global-spacing-6),
+					text-align: center,
```

Comment:
Technically this should be on the paragraph.  If you want to center align the elements, with flex/grid, we should be using `align-items`/`justify-content` notation.